builder/ebssurrogate/builder.go

```diff
@@ -66,15 +66,15 @@ type Config struct {
 	// block containing a `name` and a `value` field. In HCL2 mode the
 	// [`dynamic_block`](https://packer.io/docs/templates/hcl_templates/expressions.html#dynamic-blocks)
 	// will allow you to create those programatically.
-	VolumeRunTag config.NameValues `mapstructure:"run_volume_tag" required:"false"`
+	VolumeRunTag config.NameValues `mapstructure:"run_volume_tag"               required:"false"`
 	// what architecture to use when registering the final AMI; valid options
 	// are "arm64", "i386", "x86_64", or "x86_64_mac". Defaults to "x86_64".
-	Architecture string `mapstructure:"ami_architecture" required:"false"`
+	Architecture string `mapstructure:"ami_architecture"             required:"false"`
 	// The boot mode. Valid options are `legacy-bios` and `uefi`. See the documentation on
 	// [boot modes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ami-boot.html) for
 	// more information. Defaults to `legacy-bios` when `ami_architecture` is `x86_64` and
 	// `uefi` when `ami_architecture` is `arm64`.
-	BootMode string `mapstructure:"boot_mode" required:"false"`
+	BootMode string `mapstructure:"boot_mode"                    required:"false"`
```

Comment:
I wonder, is this a linter's doing? I'm not sure I understand what's this modification for, since it adds a bunch of spaces to almost all the config values, without clear benefit.
I'm not necessarily opposed to lint changes, but in this case I find this one distracting, could I ask you to remove those from the changes in the PR?